scripts/deploy.js

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+const {argv, env, exit} = require('process');
+const {execSync} = require('child_process');
+
+const {loadEnv} = require('./common');
+
+const stage = argv[2];
+if (!stage) {
+    console.error("Usage: deloy.js <stage name>")
```

Comment:
Do we need to recommend using node here too? `Usage: node deploy.js <stage name>` ? Since this seem to be used from inside yarn and not strictly run by the user from the terminal, this comment seem unnecessary and just a nitpick.

Reply:
Good question. I was trying to be agnostic about the working directory when writing this. I suppose we could say `node deploy.js` but then my concern would be that someone expects to be able to run that, which they can only do if pwd is `transfer/scripts`. I added the hashbang at the top of both deploy.js and uncache.js so I think it should be equivalent to if you asked bash to do `node ./scripts/deploy.js`, for example?